backend/tools/import/README.md

```diff
@@ -0,0 +1,39 @@
+# PIMS Import Tool
+
+The PIMS Import Tool provides a way to parse a JSON file containing an array of items and send bundled HTTP requests to a configured API endpoint for the purpose of importing data.
+
+This console application can send any type of array of JSON objects to any API endpoint.
+
+Use this console application to import inventory data into PIMS.
+
+## Setup
+
+Create a `.env` file and populate it with the following;
+
+```
+ASPNETCORE_ENVIRONMENT=Development
+Api__Url=http://localhost:3000/api/tools/import/properties
```

Comment:
can we use something besides 3000 here?

Reply:
I think this update will make it better.